Makefile

```diff
@@ -140,7 +140,12 @@ helm.prepare.universal-crossplane: crossplane
 
 olm.build: $(HELM) $(OLMBUNDLE)
 	@$(INFO) Generating OLM bundle
-	@$(HELM) -n upbound-system template $(HELM_CHARTS_DIR)/$(PACKAGE_NAME) --set upbound.controlPlane.permission=edit > $(WORK_DIR)/olm.yaml
+	@$(HELM) -n upbound-system template $(HELM_CHARTS_DIR)/$(PACKAGE_NAME) \
+		--set upbound.controlPlane.permission=edit \
+		--set securityContextCrossplane.runAsUser=null \
```

Comment:
Setting it to `null` removes the key completely, see https://github.com/helm/helm/pull/2648